UI/src/app/app.component.html

```diff
@@ -2,35 +2,41 @@
   <button mat-icon-button [matMenuTriggerFor]="menu" aria-label="cdm menu">
     <mat-icon>more_vert</mat-icon>
   </button>
+  <button mat-icon-button>
+    <mat-icon svgIcon="help"></mat-icon>
+  </button>
   <span style="flex: 1 1 auto"></span>
   <h1>CDM ETL</h1>
   <mat-menu #menu="matMenu">
     <input
       #sourceUpload
       type="file"
       (change)="onFileUpload($event)"
-      style="display: none"
-    />
+      style="display: none"/>
```

Comment:
What reason to use html inline styles instead of scss styles?

Reply:
It's not my code here, just formatting previous

Reply:
please move it to scss file